subgraphs/exchange-stableswap/mappings/services/swap.ts

```diff
@@ -70,6 +73,7 @@ export function swap(event: ethereum.Event, params: SwapParams): void {
 
   // update pair volume data, use tracked amount if we have it as its probably more accurate
   pair.volumeUSD = pair.volumeUSD.plus(trackedAmountUSD);
+  pair.volumeOutUSD = pair.volumeOutUSD.plus(amount0Out.times(price0)).plus(amount1Out.times(price1));
```

Comment:
The `trackedAmountUSD` above only takes tokens, which are on whitelist, into account.
Shall we follow the same logic when calculating `volumeOutUSD`?

Reply:
Also can extract a separate function and move it to `mappings/utils/pricing.ts`

Reply:
Sure, let me do some adjustment